File: backend/services/audit-service/src/main/java/com/lmspilot/audit/AuditServiceApplication.java
package com.lmspilot.audit;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;


@SpringBootApplication(scanBasePackages = "com.lmspilot")
public class AuditServiceApplication {
    public static void main(String[] args) { SpringApplication.run(AuditServiceApplication.class, args); }
}

File: backend/services/audit-service/src/main/kotlin/com/lmspilot/audit/AuditServiceApplication.kt
package com.lmspilot.audit

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["com.lmspilot"])
class AuditServiceApplication

fun main(args: Array<String>) {
    runApplication<AuditServiceApplication>(*args)
}
